### PR TITLE
Add colormap control to be used with Choropleth layer.

### PR DESCRIPTION
**Pull request by HaudinFlorence**
Add a ColormapControl class inherited from WidgetControl class. The widget used is an Output widget.

### CODE REVIEW

File: ipyleaflet/leaflet.py [hunk 1]
```diff
@@ -1866,38 +1870,6 @@ def __init__(self, legend, *args, name="Legend", **kwargs):
...
-        self.position = position
-
-    @property
-    def positionning(self):
-        return self.position
-
-    @positionning.setter
-    def positionning(self, position):
-        self.position = position
-
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py:1900]: This part should probably be done in a separate PR?

File: ipyleaflet/leaflet.py [hunk 2]
```diff
@@ -2,16 +2,19 @@
 # Distributed under the terms of the Modified BSD License.
 #
 
+from ast import In
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py:5]: Is this needed?

File: ipyleaflet/leaflet.py [hunk 3]
```diff
@@ -2,16 +2,19 @@
...
 import copy
 import asyncio
 import json
 import xyzservices
 from datetime import date, timedelta
 from math import isnan
+import branca.colormap as cm
+from branca.colormap import linear
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py:13]: This makes ipyleaflet dependent on branca now. So we either need to add it as a dependency in the `setup.py`, or we can find a way to import it lazily for the `ColormapControl` usage.

File: ipyleaflet/leaflet.py [hunk 4]
```diff
@@ -2,16 +2,19 @@
...
 
 from ipywidgets import (
     Widget, DOMWidget, Box, Color, CallbackDispatcher, widget_serialization,
-    interactive, Style
+    interactive, Style, Output, Layout
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py:17]: We probably don't need to import `Layout`?

**Review by martinRenou — CHANGES_REQUESTED** [review]
Thanks!

File: ipyleaflet/leaflet.py (hunk 2 above)
Comment (HaudinFlorence) [reply ipyleaflet/leaflet.py]: This was here. I just removed it.

Comment (martinRenou) [reply ipyleaflet/leaflet.py]: The diff says it's an addition, not a removal

File: ipyleaflet/leaflet.py (hunk 3 above)
Comment (HaudinFlorence) [reply ipyleaflet/leaflet.py:11]: I will add a dependency in the setup.py

File: ipyleaflet/leaflet.py (hunk 1 above)
Comment (HaudinFlorence) [reply ipyleaflet/leaflet.py]: Ok, this change has been removed and will be included in a separate PR.

**Comment by martinRenou** [comment]
Please update galata references

**Comment by martinRenou** [comment]
Kicking CI

File: docs/source/controls/colormap_control.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+    data_unemployment =  dict(zip(unemployment['State'].tolist(), unemployment['Unemployment'].tolist()))
+
+
+
+    layer = ipyleaflet.Choropleth(
+    geo_data=geo_json_data,
+    choro_data=data_unemployment,
+    colormap=colormap_choice,
+    style={"dashArray": "5, 5"}
```
Comment (martinRenou) [inline docs/source/controls/colormap_control.rst:42]: Missing indentation

```suggestion
        geo_data=geo_json_data,
        choro_data=data_unemployment,
        colormap=colormap_choice,
        style={"dashArray": "5, 5"}
```

File: examples/ColormapControl.ipynb [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d822ba7e-2347-4517-a285-159426c71822",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "myColormapControl = ColormapControl(\n",
```
Comment (martinRenou) [inline examples/ColormapControl.ipynb:40]: Same suggestion in the example notebook
```suggestion
    "colormap_control = ColormapControl(\n",
```
